### PR TITLE
Improved rttmon profile

### PR DESCRIPTION
**Pull request by Mesverrum**
accidentally removed this while changing env sensors, compared this to several walks for improvements

### CODE REVIEW

File: profiles/kentik_snmp/cisco/cisco-all-devices.yml [hunk 1]
```diff
@@ -671,3 +671,142 @@ metrics:
...
+            notReady: 3
+            createAndGo: 4
+            createAndWait: 5
+            destroy: 6
+        tag: rtt_operation_admin_status
+      - column:
+          OID: 1.3.6.1.4.1.9.9.42.1.2.2.1.1
+          name: rttMonEchoAdminProtocol
+          enum:
+            notApplicable(1), ipIcmpEcho(2), ipUdpEchoAppl(3), snaRUEcho(4), snaLU0EchoAppl(5), snaLU2EchoAppl(6), snaLU62Echo(7), snaLU62EchoAppl(8), appleTalkEcho(9), appleTalkEchoAppl(10), decNetEcho(11), decNetEchoAppl(12), ipxEcho(13), ipxEchoAppl(14), isoClnsEcho(15), isoClnsEchoAppl(16), vinesEcho(17), vinesEchoAppl(18), xnsEcho(19), xnsEchoAppl(20), apolloEcho(21), apolloEchoAppl(22), netbiosEchoAppl(23), ipTcpConn(24), httpAppl(25), dnsAppl(26), jitterAppl(27), dlswAppl(28), dhcpAppl(29), ftpAppl(30), mplsLspPingAppl(31), voipAppl(32), rtpAppl(33), icmpJitterAppl(34), ethernetPingAppl(35), ethernetJitterAppl(36), videoAppl(37), y1731dmm(38), y17311dm(39), y1731lmm(40), mcastJitterAppl(41), y1731slm(42), y1731dmmv1(43), fabricPathEchoAppl(44)
```
Comment (thezackm) [inline profiles/kentik_snmp/cisco/cisco-all-devices.yml:732]: @Mesverrum -  I don't think this syntax is going to work

File: profiles/kentik_snmp/cisco/cisco-all-devices.yml [hunk 2]
```diff
@@ -671,3 +671,142 @@ metrics:
...
+    table:
+      OID: 1.3.6.1.4.1.9.9.42.1.2.10
+      name: rttMonLatestRttOperTable
+    symbols:
+      - OID: 1.3.6.1.4.1.9.9.42.1.2.10.1.1
+        name: rttMonLatestRttOperCompletionTime
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.9.9.42.1.2.1.1.3
+          name: rttMonCtrlAdminTag
```
Comment (thezackm) [inline profiles/kentik_snmp/cisco/cisco-all-devices.yml:684]: this OID is deprecated; replaced with:

```
OID: 1.3.6.1.4.1.9.9.42.1.2.1.1.12
name: rttMonCtrlAdminLongTag

**Review by thezackm — CHANGES_REQUESTED** [review]

File: profiles/kentik_snmp/cisco/cisco-all-devices.yml (hunk 2 above)
Comment (Mesverrum) [reply profiles/kentik_snmp/cisco/cisco-all-devices.yml]: superceded, deprecated, tomato tamahto

File: profiles/kentik_snmp/cisco/cisco-all-devices.yml (hunk 1 above)
Comment (Mesverrum) [reply profiles/kentik_snmp/cisco/cisco-all-devices.yml]: check

**Review by thezackm — APPROVED** [review]
🚀

**Pull request merged**